mocks/test.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
Are we using this file? If not, please remove it

Reply:
👍